core/src/test/java/org/hibernate/ogm/backendtck/storedprocedures/indexed/IndexedStoredProcedureCallTest.java

```diff
@@ -0,0 +1,200 @@
+			assertThat( singleResult ).isEqualTo( 1.0d );
+		}
+		else {
+			assertThat( singleResult ).isEqualTo( 1 );
+		}
+	}
+	/**
+	 * Testing a call of stored procedure (function)
+	 * @throws Exception
+	 */
+	@Test
+	public void dynamicCallOfStoredProcedureWithParametersAndListEntityResult() throws Exception {
+
+		StoredProcedureQuery call3 = em.createStoredProcedureQuery( TEST_RESULT_SET_STORED_PROC, Car.class );
+		assertThat( call3 ).isInstanceOfAny( OgmStoredProcedureQuery.class );
+		call3.registerStoredProcedureParameter( 0,Void.class, ParameterMode.REF_CURSOR );
+		call3.registerStoredProcedureParameter( 1, String.class, ParameterMode.IN );
+		call3.registerStoredProcedureParameter( 2, String.class, ParameterMode.IN );
+		call3.setParameter( 1, "id" );
+		call3.setParameter( 2, "title" );
+		List<Car> listResult = call3.getResultList();
+		assertThat( listResult ).isNotNull();
+		assertThat( listResult.size() ).isEqualTo( 1 );
+		assertThat( listResult.get( 0 ) ).isEqualTo( new Car( "id","title" ) );
+
+		StoredProcedureQuery call4 = em.createStoredProcedureQuery( TEST_RESULT_SET_STORED_PROC, "carMapping" );
+		assertThat( call4 ).isInstanceOfAny( OgmStoredProcedureQuery.class );
+		call4.registerStoredProcedureParameter( 0,Void.class, ParameterMode.REF_CURSOR );
+		call4.registerStoredProcedureParameter( 1, String.class, ParameterMode.IN );
+		call4.registerStoredProcedureParameter( 2, String.class, ParameterMode.IN );
+		call4.setParameter( 1, "id1" );
+		call4.setParameter( 2, "title1" );
+		listResult = call4.getResultList();
+		assertThat( listResult ).isNotNull();
+		assertThat( listResult.size() ).isEqualTo( 1 );
+		assertThat( listResult.get( 0 ) ).isEqualTo( new Car( "id1","title1" ) );
+	}
+
+	@Test
+	public void staticCallOfStoredProcedureWithParametersAndListEntityResult() throws Exception {
+		//function with one parameter and returned entity
+
+
+		StoredProcedureQuery call3 = em.createNamedStoredProcedureQuery( "testproc4_1" );
+		assertThat( call3 ).isInstanceOfAny( OgmStoredProcedureQuery.class );
+		call3.setParameter( 1, "id" );
+		call3.setParameter( 2, "title" );
+		List<Car> listResult = call3.getResultList();
+		assertThat( listResult ).isNotNull();
+		assertThat( listResult.size() ).isEqualTo( 1 );
+		assertThat( listResult.get( 0 ) ).isEqualTo( new Car( "id", "title" ) );
+
+		StoredProcedureQuery call4 = em.createNamedStoredProcedureQuery( "testproc4_2" );
+		assertThat( call4 ).isInstanceOfAny( OgmStoredProcedureQuery.class );
+		call4.setParameter( 1, "id1" );
+		call4.setParameter( 2, "title1" );
+		listResult = call4.getResultList();
+		assertThat( listResult ).isNotNull();
+		assertThat( listResult.size() ).isEqualTo( 1 );
+		assertThat( listResult.get( 0 ) ).isEqualTo( new Car( "id1", "title1" ) );
+	}
+
+	@Override
+	protected void configure(GetterPersistenceUnitInfo info) {
+
+		Properties properties = info.getProperties();
+
+		if ( TestHelper.getCurrentDialectType().equals( GridDialectType.HASHMAP ) ) {
```

Comment:
Let's avoid this, please. Let's simply skip this tests for the dialects that don't support storedProcedures

Reply:
You can simply create a separate test in the package `org.hibernate.ogm.backendtck` that uses IndexedStoredProcProvider

Reply:
fixed